next.config.mjs

```diff
@@ -1,4 +1,10 @@
+import { createRequire } from 'module'
+const require = createRequire(import.meta.url)
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+	enabled: process.env.ANALYZE === 'true',
+})
+
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {}
 
-export default nextConfig;
+export default withBundleAnalyzer(nextConfig)
```

Comment:
Those changes are common for all the PRs

Reply:
I see but is this something bad ? Im not sure I understand the problem. Everytime after I made a PR and wanna make new branch I just type in console "git checkout -b branch-name" and then after some work I stage and commit all files

Reply:
- When the same changes appear in multiple PRs, reviewers have to examine the same code snippets repeatedly. This can lead to fatigue and potentially overlooking issues.
- If the common changes require fixes, they must be implemented in multiple places, increasing the risk of errors and inconsistencies.
- When several PRs contain the same changes, conflicts may arise during merging, especially if PRs are merged in different orders.